operator/webserver.py

```diff
@@ -1,31 +1,49 @@
 #!/usr/bin/env python
 
-from flask import Flask, request, abort
-import subprocess, time, signal, sys, os, airsim
+from flask import Flask, request, abort, render_template
+import subprocess, time, signal, sys, os, json
+from datetime import datetime
+
+import sys
+sys.path.append('../AirSim/PythonClient')
+from airsim.client import CarClient
```

Comment:
This is an unsexy way to import a local package, bit I couldn't get it to work using 'proper' __init__.py's

Reply:
I think this is still looks beautiful :heart_eyes: